migrate/src/migrations/mig_20250617.rs

```diff
@@ -0,0 +1,253 @@
+use fiber::store::Store;
+use fiber::{store::migration::Migration, Error};
+use fiber_v061::fiber::types::Pubkey;
+use fiber_v061::store::store_impl::StoreKeyValue;
+use indicatif::ProgressBar;
+use std::sync::Arc;
+use tracing::info;
+
+const MIGRATION_DB_VERSION: &str = "20250617093908";
+
+pub use fiber_v051::fiber::channel::ChannelActorState as OldChannelActorState;
+pub use fiber_v051::fiber::channel::PendingTlcs as OldPendingTlcs;
+pub use fiber_v051::fiber::channel::TlcInfo as OldTlcInfo;
+pub use fiber_v051::fiber::channel::TlcState as OldTlcState;
+pub use fiber_v051::fiber::graph::PaymentSession as OldPaymentSession;
+pub use fiber_v051::fiber::graph::PaymentSessionStatus as OldPaymentSessionStatus;
+pub use fiber_v051::fiber::graph::SessionRoute as OldSessionRoute;
+pub use fiber_v051::fiber::graph::SessionRouteNode as OldSessionRouteNode;
+pub use fiber_v051::fiber::network::HopHint as OldHopHint;
+pub use fiber_v051::fiber::network::SendPaymentData as OldPaymentData;
+pub use fiber_v051::fiber::types::Hash256 as OldHash256;
+
+pub use fiber_v061::fiber::channel::ChannelActorState as NewChannelActorState;
+pub use fiber_v061::fiber::channel::PendingTlcs as NewPendingTlcs;
+pub use fiber_v061::fiber::channel::TlcInfo as NewTlcInfo;
+pub use fiber_v061::fiber::channel::TlcState as NewTlcState;
+pub use fiber_v061::fiber::graph::Attempt;
+pub use fiber_v061::fiber::graph::PaymentSession as NewPaymentSession;
+pub use fiber_v061::fiber::graph::PaymentSessionStatus as NewPaymentSessionStatus;
+pub use fiber_v061::fiber::graph::SessionRoute as NewSessionRoute;
+pub use fiber_v061::fiber::graph::SessionRouteNode as NewSessionRouteNode;
+pub use fiber_v061::fiber::network::HopHint as NewHopHint;
+pub use fiber_v061::fiber::network::SendPaymentData as NewPaymentData;
+pub use fiber_v061::fiber::types::Hash256 as NewHash256;
+pub use fiber_v061::store::store_impl::KeyValue as NewKeyValue;
+
+pub struct MigrationObj {
+    version: String,
+}
+
+impl Default for MigrationObj {
+    fn default() -> Self {
+        Self::new()
+    }
+}
+
+impl MigrationObj {
+    pub fn new() -> Self {
+        Self {
+            version: MIGRATION_DB_VERSION.to_string(),
+        }
+    }
+}
+
+impl Migration for MigrationObj {
+    fn migrate<'a>(
+        &self,
+        db: &'a Store,
+        _pb: Arc<dyn Fn(u64) -> ProgressBar + Send + Sync>,
+    ) -> Result<&'a Store, Error> {
+        info!(
+            "MigrationObj::migrate to {} ...........",
+            MIGRATION_DB_VERSION
+        );
+
+        // NodeAnnouncement is changed, we need to delete the old broadcast messages
+        info!("clear broadcast messages ...");
+        delete_broadcast_messages(db);
```

Comment:
we now use `load_from_store` to build network graph when starting a node, so if we delete all boradcast messages, network graph will be empty.

https://github.com/nervosnetwork/fiber/blob/a3e022e67d245992089f62775b2e1c88aa9fc857/crates/fiber-lib/src/fiber/graph.rs#L628-L638

I'm not sure what the consequence of this.